lib/galaxy/webapps/galaxy/api/tool_dependencies.py

```diff
@@ -193,30 +193,37 @@ def summarize_toolbox(self, trans: ProvidesAppContext, **kwds):
         """
         GET /api/dependency_resolvers/toolbox
 
-        Summarize requirements across toolbox (for Tool Management grid). This is an experiemental
-        API particularly tied to the GUI - expect breaking changes until this notice is removed.
+        Summarize requirements and their dependencies across toolbox (for Tool Management grid).
+        This is an experimental API particularly tied to the GUI - expect
+        breaking changes until this notice is removed.
 
         Container resolution via this API is especially experimental and the container resolution
         API should be used to summarize this information instead in most cases.
 
+        Note that for container resolution by default resolvers are skipped that require online access
+        or may build/cache a container, i.e. only resolvers whose name starts with `"cached"`,`"explicit"`,
+          or `"fallback"` are used
+
         :type   index:    int
         :param  index:    index of the dependency resolver
         :type   tool_ids: str
-        :param  tool_ids: tool_id to install dependency for
+        :param  tool_ids: comma separated tool_ids to summarize dependencies for
```

Comment:
```suggestion
        :param  tool_ids: comma separated tool_ids to summarize dependencies for
```
or a list in the request body ... which we should prefer.

Reply:
> or a list in the request body

Is this possible in a GET request?